src/extensions/displayExtensionInfo.ts

```diff
@@ -18,33 +29,77 @@ const TASKS_ROLE = "cloudtasks.enqueuer";
 const TASKS_API = "cloudtasks.googleapis.com";
 
 /**
- * displayExtInfo prints the extension info displayed when running ext:install.
+ * Displays info about an extension version, whether it is uploaded to the registry or a local spec.
  *
- * @param extensionName name of the extension to display information about
- * @param spec extension spec
- * @param published whether or not the extension is a published extension
- */
-export async function displayExtInfo(
-  extensionName: string,
-  publisher: string,
+ * @param spec the extension spec
+ * @param extensionVersion the extension version
+ * */
+export async function displayExtensionVersionInfo(
   spec: ExtensionSpec,
-  published = false
+  extensionVersion?: ExtensionVersion,
+  latestApprovedVersion?: string,
+  latestVersion?: string
 ): Promise<string[]> {
-  const lines = [];
-  lines.push(`**Name**: ${spec.displayName}`);
-  if (publisher) {
-    lines.push(`**Publisher**: ${publisher}`);
-  }
+  const lines: string[] = [];
+  const extensionRef = extensionVersion
+    ? refs.toExtensionRef(refs.parse(extensionVersion?.ref))
+    : "";
+  lines.push(
+    `${clc.bold("Extension:")} ${spec.displayName ?? "Unnamed extension"} ${
+      extensionRef ? `(${extensionRef})` : ""
+    }`
+  );
   if (spec.description) {
-    lines.push(`**Description**: ${spec.description}`);
+    lines.push(`${clc.bold("Description:")} ${spec.description}`);
+  }
+  let versionNote = "";
+  const latestRelevantVersion = latestApprovedVersion || latestVersion;
+  if (latestRelevantVersion && semver.eq(spec.version, latestRelevantVersion)) {
+    versionNote = `- ${clc.green("Latest")}`;
+  }
+  if (extensionVersion?.state === "DEPRECATED") {
+    versionNote = `- ${clc.red("Deprecated")}`;
   }
-  if (published) {
-    if (spec.license) {
-      lines.push(`**License**: ${spec.license}`);
+  lines.push(`${clc.bold("Version:")} ${spec.version} ${versionNote}`);
+  if (extensionVersion) {
+    let reviewStatus: string;
+    switch (extensionVersion.listing?.state) {
+      case "APPROVED":
+        reviewStatus = clc.bold(clc.green("Accepted"));
+        break;
+      case "REJECTED":
+        reviewStatus = clc.bold(clc.red("Rejected"));
+        break;
+      default:
+        reviewStatus = clc.bold(clc.yellow("Unreviewed"));
+        break;
```

Comment:
nit: break is unnecessary on the last block of a switch statement, especially since this is a default block (and should always be last)